sdk/storage/test-resources-post.ps1

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+param (
+    [hashtable] $DeploymentOutputs,
+    [string] $TenantId,
+    [string] $TestApplicationId,
+    [string] $TestApplicationSecret
+)
+
+# outputs from the ARM deployment passed in from New-TestResources
+$outputs = $PSBoundParameters['DeploymentOutputs']
+$PrimaryAccountName = $DeploymentOutputs['PRIMARY_STORAGE_ACCOUNT_NAME']
+$PrimaryAccountKey = $DeploymentOutputs['PRIMARY_STORAGE_ACCOUNT_KEY']
+$SecondaryAccountName = $DeploymentOutputs['SECONDARY_STORAGE_ACCOUNT_NAME']
+$SecondaryAccountKey = $DeploymentOutputs['SECONDARY_STORAGE_ACCOUNT_KEY']
+$PremiumAccountName = $DeploymentOutputs['PREMIUM_STORAGE_ACCOUNT_NAME']
+$PremiumAccountKey = $DeploymentOutputs['PREMIUM_STORAGE_ACCOUNT_KEY']
+$DataLakeAccountName = $DeploymentOutputs['DATALAKE_STORAGE_ACCOUNT_NAME']
+$DataLakeAccountKey = $DeploymentOutputs['DATALAKE_STORAGE_ACCOUNT_KEY']
+
+# AAD parameters passed in from New-TestResources
+$AAdTenantId = $PSBoundParameters['TenantId']
+$AadApplicationId = $PSBoundParameters['TestApplicationId']
+$AadApplicationSecret = $PSBoundParameters['TestApplicationSecret']
+
+# Construct the content of the configuration file that the Storage tests expect
+$content = 
+"<TestConfigurations>
+  <TargetTestTenant>ProductionTenant</TargetTestTenant>

```

Comment:
Is this temporary?

Reply:
Eventually we should move away from requiring the config file, but I figured that could be addressed separately.

Reply:
Should we just do it?

Reply:
I don't think we want to do this until we figure out an end-to-end solution for running live tests locally without the config file.